config.json

```diff
@@ -1056,6 +1056,18 @@
       "uuid": "0c17444e-65b0-4bc8-a7e1-ee6ab8c2e078"
     },
     {
+      "core": false,
+      "difficulty": 4,
```

Comment:
I think this is more a 2 or 3 on difficulty? As far as ciphers go this is a starting point.